src/components/HPEDevFooter/index.js

```diff
@@ -0,0 +1,55 @@
+import React, { useContext } from 'react';
+import { Box, Text, Image, Header, ResponsiveContext, Button } from 'grommet';
+
+export const HPEDevFooter = () => {
+  const footerLinks = [
+    { label: 'Platforms', href: 'https://developer.hpe.com/platforms' },
+    { label: 'Open Source', href: 'https://developer.hpe.com/projects' },
+    { label: 'Community', href: 'https://developer.hpe.com/community' },
+    { label: 'Events', href: 'https://developer.hpe.com/events' },
+  ];
+  const size = useContext(ResponsiveContext);
+  //========================================= Footer
+  return (
+    <Header
+      align="center"
+      direction="row"
+      flex={false}
+      justify="between"
+      gap="medium"
+      fill="horizontal"
+      pad="xsmall"
```

Comment:
This pad should be larger the sides are 48px so `horizontal: 'large'` then check with figma for vertical I believe medium should work